TempleEight/docs/guide/orchestration.md

```diff
@@ -3,3 +3,174 @@ id: orchestration
 title: Orchestration
 sidebar_label: Orchestration
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Considering how your application components are orchestrated is important as the components themselves.
+Temple provides several industry standards methods of automatically deploying your services.
+
+## Adding Orchestration to your Templefile
+l
+Orchestrating your project automatically requires only a single line of code in your Templefile.
+Working from the example in the [Getting Started](../getting-started) guide:
+
+```templefile {4}
+ExampleProject: project {
+  #language(go);
+  #database(postgres);
+  #provider(dockerCompose);
+}
+
+ExampleService: service {
+  foo: string;
+  bar: int;
+}
+```
+We can see here that the Templefile features a `#provider` annotation in the project block.
+This is how we specify that our project uses orchestration, the name in the argument specifies which framework we are generating for.
+
+The currently supported frameworks are:
+* Docker Compose
+* Kubernetes
+
+All of the frameworks are generated to share a common toolchain, so that usage is the same no matter which you choose.
+
+## Kong API Gateway
```

Comment:
Hmm this structure feels kinda wrong. You're talking about deployment, and then go on to talk about API gateways?

I'd personally bring Docker Compose and Kube up, then switch this after, explaining that it's used in both instances